roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -65,16 +69,14 @@ def run(self, tmp, task_vars):
 
     def required_images(self, task_vars):
         deployment_type = get_var(task_vars, "deployment_type")
-        # FIXME(juanvallejo): we should handle gracefully with a proper error
-        # message when given an unexpected value for `deployment_type`.
-        image_base_name = self.docker_image_base[deployment_type]
+        if deployment_type not in self.docker_image_base:
+            msg = "Unable to determine required images for deployment type {deployment}"
+            raise OpenShiftCheckException(msg.format(deployment=deployment_type))
```

Comment:
I think it would be consistent with cases such as [this one](https://github.com/openshift/openshift-ansible/pull/3518/files#diff-dd45df49348ab2716e213a0ecc5b0f4dR24) that we skip the check if a deployment type is not supported